### DISCUSSION TITLE
SplineEvent to have all infos of the target

### DISCUSSION THREAD

**Discussion by Prasanna-kumar-Patra** [splinetool/react-spline#105] [Q&A]
SplineEvent to have all infos of the target
like parent info,  position and coordinates event type and  all that we get in normal DOM level events

**Comment by marcofugaro** [comment]
You can get the event object info like this:

```js
import Spline from '@splinetool/react-spline';

export default function App() {
  const spline = useRef();

  function onLoad(splineApp) {
    spline.current = splineApp;
  }

  function onMouseDown(e) {
    const obj = spline.current.findObjectById(e.target.id)
    console.log(obj) // it has position, parent ecc..
  }

  return (
    <div>
      <Spline
        scene="https://prod.spline.design/6Wq1Q7YGyM-iab9i/scene.splinecode"
        onLoad={onLoad}
        onMouseDown={onMouseDown}
      />
    </div>
  );
}
```

**Reply by Prasanna-kumar-Patra** [reply]
Hi Thanks for the reply
I am expecting all properties that are in the property panel of the editor
Like on/off physics and read and write the physics props also
<img width="227" alt="image" src="https://user-images.githubusercontent.com/5335912/211257253-40c6a745-6a3c-4c08-9726-f56fcb8c8f50.png">

**Reply by Prasanna-kumar-Patra** [reply]
anyone able to access and set physics properties

**Reply by Prasanna-kumar-Patra** [reply]
any idea if this is possible to get/set physics props via react that are available in editor
